### PR TITLE
refactor: unify {gw, ckb}-types and {gw, ckb}-jsonrpc-types

### PR DESCRIPTION
**Pull request by blckngm**

### CODE REVIEW

File: gwos/crates/types/src/conversion/blockchain.rs [hunk 1]
```diff
@@ -60,11 +59,18 @@ impl_conversion_for_vector!(Bytes, BytesVec, BytesVecReader);
...
 impl_conversion_for_packed_iterator_pack!(CellInput, CellInputVec);
-impl_conversion_for_packed_iterator_pack!(UncleBlock, UncleBlockVec);
+// impl_conversion_for_packed_iterator_pack!(UncleBlock, UncleBlockVec);
 impl_conversion_for_packed_iterator_pack!(Byte32, Byte32Vec);
+
+#[cfg(feature = "std")]
+pub fn cap_bytes(len: usize) -> ckb_types::core::Capacity {
+    use ckb_types::core::Capacity;
+
+    Capacity::bytes(len).unwrap_or_else(|_| Capacity::shannons(u64::MAX))
```
Comment (jjyr) [inline gwos/crates/types/src/conversion/blockchain.rs]:
```suggestion
    Capacity::bytes(len).unwrap()
```

`u64::MAX` may cause unexpected behaviors. Use `unwrap` or return a Result

**Review by jjyr — DISMISSED** [review]

File: gwos/crates/types/src/conversion/blockchain.rs (hunk 1 above)
Comment (blckngm) [reply gwos/crates/types/src/conversion/blockchain.rs]: Please see the latest commit.

Comment (jjyr) [reply gwos/crates/types/src/conversion/blockchain.rs]: LGTM

**Review by jjyr — APPROVED** [review]

**Review by Flouse — APPROVED** [review]

**Comment by Flouse** [comment]
https://github.com/godwokenrises/godwoken/actions/runs/4034785042/jobs/6936264110#step:6:848
One check failed.
@sopium

**Comment by blckngm** [comment]
> https://github.com/godwokenrises/godwoken/actions/runs/4034785042/jobs/6936264110#step:6:848 One check failed. @sopium

I think it's because web3-indexer Dockerfile and this workflow need to adapt to using path dependencies.

**Pull request merged**